### ISSUE TITLE
Version/WEB and FTL not showing up

### ISSUE THREAD

**Issue by mnefk** [hassio-addons/addon-pi-hole#19]
# Problem/Motivation

> Pi-hole Version N/A Web Interface Version N/A FTL Version N/A

## Steps to reproduce

> Restarted Add-On, no change

**Comment by frenck** [comment]
Interesting, since I'm not able to reproduce it.
Could you provide a debug log, your add-on configuration and describe the type of setup you are using?

**Comment by mnefk** [comment]
here from Add-On Log:

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] 01-resolver-resolv: applying... 
[fix-attrs.d] 01-resolver-resolv: exited 0.
[fix-attrs.d] 01-sudo: applying... 
[fix-attrs.d] 01-sudo: exited 0.
[fix-attrs.d] 02-nginx: applying... 
[fix-attrs.d] 02-nginx: exited 0.
[fix-attrs.d] 03-pihole: applying... 
[fix-attrs.d] 03-pihole: exited 0.
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 

-----------------------------------------------------------
 Hass.io Add-on: Pi-hole v0.4.0

 Network-wide ad blocking using your Hass.io instance

 From: Community Hass.io Add-ons
 By: Franck Nijhof <frenck@addons.community>
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
Log level is set to INFO
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] 02-updates.sh: executing... 

INFO: You are running the latest version of this add-on
[cont-init.d] 02-updates.sh: exited 0.
[cont-init.d] 10-requirements.sh: executing... 
[cont-init.d] 10-requirements.sh: exited 0.
[cont-init.d] 11-patches.sh: executing... 

[cont-init.d] 11-patches.sh: exited 0.
[cont-init.d] 11-pihole.sh: executing... 
[cont-init.d] 11-pihole.sh: exited 0.
[cont-init.d] 12-dnsmasq.sh: executing... 
[cont-init.d] 12-dnsmasq.sh: exited 0.
[cont-init.d] 13-nginx.sh: executing... 
[cont-init.d] 13-nginx.sh: exited 0.
[cont-init.d] 14-password.sh: executing... 
  [✓] New password set
[cont-init.d] 14-password.sh: exited 0.
[cont-init.d] 15-hostname.sh: executing... 

[cont-init.d] 15-hostname.sh: exited 0.
[cont-init.d] 16-logfiles.sh: executing... 
[cont-init.d] 16-logfiles.sh: exited 0.
[cont-init.d] 17-hosts.sh: executing... 
[cont-init.d] 17-hosts.sh: exited 0.
[cont-init.d] 50-gravity.sh: executing... 
  [i] Neutrino emissions detected...
  [i] Pulling blocklist source list into range...

  [✓] Pulling blocklist source list into range

  [i] Target: raw.githubusercontent.com (hosts)
  [i] Status: Pending...

  [✓] Status: Retrieval successful

  [i] Target: mirror1.malwaredomains.com (justdomains)
  [i] Status: Pending...

  [✓] Status: No changes detected

  [i] Target: sysctl.org (hosts)
  [i] Status: Pending...

  [✓] Status: No changes detected

  [i] Target: zeustracker.abuse.ch (blocklist.php?download=domainblocklist)
  [i] Status: Pending...

  [✓] Status: No changes detected

  [i] Target: s3.amazonaws.com (simple_tracking.txt)
  [i] Status: Pending...

  [✓] Status: No changes detected

  [i] Target: s3.amazonaws.com (simple_ad.txt)
  [i] Status: Pending...

  [✓] Status: No changes detected

  [i] Target: hosts-file.net (ad_servers.txt)
  [i] Status: Pending...

  [✓] Status: No changes detected

  [i] Target: raw.githubusercontent.com (spy.txt)
  [i] Status: Pending...
  
  [✓] Status: Retrieval successful

  [i] Target: v.firebog.net (Easylist.txt)
  [i] Status: Pending...

  [✓] Status: No changes detected

  [i] Consolidating blocklists...

  [✓] Consolidating blocklists
  [i] Extracting domains from blocklists...

  [✓] Extracting domains from blocklists
  [i] 140091 domains being pulled in by gravity
  [i] Removing duplicate domains...

  [✓] Removing duplicate domains
  [i] 114610 unique domains trapped in the Event Horizon

  [i] Adding 7 blocklist source domains to the whitelist...

  [✓] Adding 7 blocklist source domains to the whitelist
  [i] Whitelisting 13 domains...

  [✓] Whitelisting 13 domains
  [i] Parsing domains into hosts format...

  [✓] Parsing domains into hosts format
  [i] Cleaning up stray matter...

  [✓] Cleaning up stray matter

  [✗] dnsmasq: no process found
pihole-FTL: no process found
  [✓] DNS service is running
  [✓] Pi-hole blocking is Enabled
[cont-init.d] 50-gravity.sh: exited 0.
[cont-init.d] 99-tests.sh: executing... 
dnsmasq: syntax check OK.
[11-Dec-2017 09:56:56] NOTICE: configuration file /etc/php7/php-fpm.conf test is successful

nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
nginx: configuration file /etc/nginx/nginx.conf test is successful
[cont-init.d] 99-tests.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
starting version 3.2.4
[services.d] done.
```


If i click debug log i will receive the following and i cant access pi-hole anymore:
`
504 Gateway Time-out
nginx/1.12.2
`

Config:
{
  "log_level": "info",
  "password": "xxxxxxx",
  "update_lists_on_start": true,
  "web_port": 80,
  "dns_port": 53,
  "ssl": false,
  "certfile": "fullchain.pem",
  "keyfile": "privkey.pem",
  "interface": "",
  "ipv6": false,
  "ipv4_address": "",
  "ipv6_address": "",
  "virtual_host": "xxxxxxx",
  "hosts": []
}

since 0.4:
![image](https://user-images.githubusercontent.com/22143075/33823325-049de3fc-de5b-11e7-9ef6-3942d2add4fd.png)

before all was fine :-)

If iam the only one with this problem after update, i just can clean it up and start with a fresh one.

**Comment by kevingrubbs** [comment]
I'm seeing the same thing on my upgrade. I upgraded from 0.1.0 to 0.4.0, but at the bottom of the screen I see:
**Pi-hole Version** N/A **Web Interface Version** N/A **FTL Version** N/A

**Comment by frenck** [comment]
I've been able to reproduce this! Not sure yet how to fix it though.
So officially marking it as an bug that needs some work.

**Comment by SouzaaThales** [comment]
I've this problem now too. Pi Hole stoped work to me.

**Comment by moonreef** [comment]
Same, I updated and now my pihole is broken.

**Comment by bcutter** [comment]
Same " Pi-hole Version N/A Web Interface Version N/A FTL Version N/A " status since version **before** 3.2.1 / 2.13. Now freshly updated to...
```
Pi-hole version is v3.2.1 (Latest: v3.2.1)
AdminLTE version is v3.2.1 (Latest: v3.2.1)
FTL version is v2.13 (Latest: v2.13)
```
... and still N/A on status page. Looking forward to a fix. Thank you guys for your great work by the way.

**Comment by frenck** [comment]
Issue is identified and is caused by a previous upgrade path.
The next release of this add-on will provide a fix for this.

**Comment by frenck** [comment]
Fixed in master, will be released in v0.5.0
